src_python/habitat_sim/utils/blender_load_replay.py

```diff
@@ -0,0 +1,233 @@
+try:
+    import bpy
+except ImportError:
+    raise ImportError(
+        "Failed to import Blender modules. This script can't run "
+        "standalone. Run `blender --python path/to/blender_load_replay.py ...`. Watch the terminal for "
+        "debug/error output."
+    )
+
+import argparse
+import json
+import os
+from dataclasses import dataclass
+
+
+@dataclass
```

Comment:
We already require attrs and it has some features dataclasses don't, any reason we aren't using them?

Reply:
The Blender python environment doesn't include `attr`. Keep in mind this script must run stand-alone inside Blender. We have no control over the environment, e.g. we can't install dependencies.